src/main/java/uk/gov/companieshouse/mapper/CompanyClosableMapper.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.companieshouse.mapper;
+
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.model.company.CompanyProfileApi;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Service
+public class CompanyClosableMapper {
+    private String[] closableTypes = new String[]{"ltd", "plc"};
```

Comment:
any chance these can be enums? Also use Lists instead of an Array i.e.
```
private static final List<String> CLOSABLE_TYPES = Arrays.asList(CompanyType.LTD, CompanyType.PLC);
```